src/parser/test/ScannerTest.cpp

```diff
@@ -374,6 +374,9 @@ TEST(Scanner, Basic) {
         CHECK_SEMANTIC_VALUE("\"\\\\\\\110 \"", TokenType::STRING, "\\H "),
         CHECK_SEMANTIC_VALUE("\"\\\\\\\\110 \"", TokenType::STRING, "\\\\110 "),
         CHECK_SEMANTIC_VALUE("\"\\\\\\\\\110 \"", TokenType::STRING, "\\\\H "),
+
+
+        CHECK_SEMANTIC_VALUE("\"己所不欲，勿施于人\"", TokenType::STRING, "己所不欲，勿施于人"),
```

Comment:
before the fix, does this sentence make service crash?
My point is can you re-produce the problem. I'm not sure the bug is due to non-asiic code

Reply:
It's another story. You could refer to discussions in #562 for details.